docs/openapi.json

```diff
@@ -157,8 +206,7 @@
             "description": "Request succeeded. The file has been added to the workspace.",
             "examples": {
               "application/json": {
-                "message": "Workflow has been added to the workspace",
-                "workflow_id": "cdcf48b1-c2f3-4693-8230-b066e088c6ac"
+                "message": "The file input.txt has been successfully transferred."
```

Comment:
Better `input.csv` or `input.dat` instead of `.txt`... but we can use some consistent examples across REANA, see `mydata.csv`, `mycode.py` etc used in musings elsewhere.